doc_comments_ai/utils.py

```diff
@@ -98,6 +126,58 @@ def extract_content_from_markdown_code_block(markdown_code_block) -> str:
     else:
         return markdown_code_block.strip()
 
+# This function retrieves the comment pattern for a specified programming language
+def get_comments_pattern_for_language(language):
+    comment_patterns = {
+        "python": r"#.*",
```

Comment:
Here I would prefer to use the value of the enum `constants.Language` as key

Reply:
Okay cool!